File: 3232/main.go
package main

import (
	"fmt"
)

func main() {
	fmt.Println(canAliceWin([]int{5,5,5,25}))
	fmt.Println(canAliceWin([]int{1,2,3,4,10}))
	fmt.Println(canAliceWin([]int{1,2,3,4,5,14}))
}

func canAliceWin(nums []int) bool {
	sum1, sum2 := 0, 0
	for _, v := range nums {
		if v < 10 {
			sum1 += v
		} else {
			sum2 += v
		}
	}
	return sum1 != sum2
}

File: 3232/main.rs
struct Solution {}

impl Solution {
    pub fn can_alice_win(nums: Vec<i32>) -> bool {
        let mut s1 = 0;
        let mut s2 = 0;
        for &v in nums.iter() {
            if v < 10 {
                s1 += v;
            } else {
                s2 += v;
            }
        }
        return s1 != s2;
    }
}

fn main() {
    println!("{}", Solution::can_alice_win(vec![5,5,5,25]));
}